barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.test.cpp

```diff
@@ -46,7 +46,6 @@ class ClientIVCTests : public ::testing::Test {
     {
```

Comment:
The comments on this function may be a bit confusing/misleading - maybe we can change to say something like "default gates added to avoid nonzero polynomials will bump size to next power of 2"